gcp/modules/couchdb/main.tf

```diff
@@ -2,17 +2,20 @@ terraform {
   backend "gcs" {}
 }
 
+variable "secrets_dir" {}
 variable "values_dir" {}
 variable "release_namespace" {
 	default = "gpii"
 }
 
 module "couchdb" {
-  source = "/exekube-modules/helm-template-release"
+  source           = "/exekube-modules/helm-release"
+  tiller_namespace = "kube-system"
+  client_auth      = "${var.secrets_dir}/kube-system/helm-tls"
 
   release_name      = "couchdb"
   release_namespace = "${var.release_namespace}"
-  release_values    = "${var.values_dir}/couchdb.yaml"
+  release_values    = "../../../../../${var.values_dir}/couchdb.yaml"
```

Comment:
Looks like `var.values_dir` is an absolute path? What's going on here?

Reply:
For helm-release, it is relative path:
https://github.com/exekube/exekube/blob/master/modules/helm-release/main.tf#L46

Reply:
Hmm, ok.

Should there be a change to https://github.com/natarajaya/gpii-infra/blob/upstream_charts/gcp/docker-compose.yaml#L34 in addition to / instead of this change?

Reply:
Alright then:
https://github.com/gpii-ops/exekube/pull/5